projects/ngrx.io/content/guide/eslint-plugin/rules/prefer-protected-state.md

```diff
@@ -18,25 +18,21 @@ This rule ensures that state changes are only managed by the Signal Store to pre
 Examples of **incorrect** code for this rule:
 
 ```ts
-// SUGGESTION ❗
```

Comment:
As mentioned in the referenced PR, this is removed to keep things consistent across the different rules.